src/ONYXKEYS.ts

```diff
@@ -731,6 +731,9 @@ const ONYXKEYS = {
 
         /** SAML login metadata for a domain */
         SAML_METADATA: 'saml_metadata_',
+
+        /** Stores domain admin account ID */
+        DOMAIN_ADMIN_PERMISSIONS: 'expensify_adminPermissions_',
```

Comment:
This is how we call it in BE, I think it will be easier if we keep it that way 
```suggestion
        EXPENSIFY_ADMIN_ACCESS_PREFIX: 'expensify_adminPermissions_',
```